flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/codegen/OperatorCodeGenerator.scala

```diff
@@ -130,6 +133,7 @@ object OperatorCodeGenerator extends Logging {
       endInputCode2: String,
       input1Type: InternalType,
       input2Type: InternalType,
+      config: TableConfig,
```

Comment:
It is not a bug... You can get config from `CodeGeneratorContext.tableConfig`.
`generateOneInputStreamOperator` have `config` parameter is redundant.

Reply:
I see. Thanks for the clarification. So the config parameter in generateOneInputStreamOperator is also redundant, right?